recipes/spirv-headers/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class SpirvheadersConan(ConanFile):
+    name = "spirv-headers"
+    homepage = "https://github.com/KhronosGroup/SPIRV-Headers"
+    description = "SPIRV-Headers"
+    topics = ("conan", "sprv", "vulkan")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+
+    license = "BSD"
```

Comment:
```suggestion
    license = "MIT-KhronosGroup"
```

This is not SPDX conform, but probably more accurate than other possible values